src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -281,7 +281,7 @@ void EnBigpo_LowerCutsceneSubCamera(EnBigpo* this, PlayState* play) {
 }
 
 void EnBigpo_InitWellBigpo(EnBigpo* this) {
-    this->actor.flags &= ~ACTOR_FLAG_1; // targetable OFF
+    this->actor.flags &= ~ACTOR_FLAG_TARGETABLE; // targetable OFF
```

Comment:
```suggestion
    this->actor.flags &= ~ACTOR_FLAG_TARGETABLE;
```
These comments can go now that the flag is named. Although we can always clean these up after the PR as well